module/lezioni.py

```diff
@@ -10,16 +10,31 @@
 
 locale.setlocale(locale.LC_ALL, 'it_IT.utf8')
 
+def get_numero_giorno(giorno):
```

Comment:
`giorno` can be `Lunedì` or `Lunedi`?
Could you make some conversion to lowercase and from `ì` to `i` or vice versa?

Reply:
Actually, giorno is always processed replacing ì with i and lowercased(?) when processing args:

`args = [str(x).lower().replace('ì', 'i') for x in args if len(x) > 2]`

If we suppose user inputs /lezioni LUNEdì, the above will transform this into lunedi.